### PR TITLE
add pyeit (a python package for electrical impedance tomography)

### PR DESCRIPTION
**Pull request by liubenyuan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by liubenyuan** [comment]
I am grad to be the maintainer of this recipe.

**Comment by liubenyuan** [comment]
@conda-forge/staged-recipes ready for review.

**Comment by liubenyuan** [comment]
@conda-forge/help-python ready for review

File: recipes/pyeit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - pyeit
+  source_files:
+    - tests/*.py
+  commands:
+    - pytest tests/
+  requires:
+    - pytest
+    - pytest-cov
+    - nose
```
Comment (beenje) [inline recipes/pyeit/meta.yaml:39]: You only use `pytest` to run the tests.
`nose` isn't needed and you don't need `pytest-cov` either.

File: recipes/pyeit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+about:
+  home: https://github.com/liubenyuan/pyEIT
+  summary: A Python based framework for EIT (Electrical Impedance Tomography)
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - byliu
```
Comment (beenje) [inline recipes/pyeit/meta.yaml:49]: Isn't your GitHub username `liubenyuan`?

File: recipes/pyeit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - pandas
+    - python >=3.6
+    - scipy
+    - matplotlib-base
+    - shapely
```
Comment (beenje) [inline recipes/pyeit/meta.yaml:27]: This looks ok to me, but note that it doesn't match the `setup.cfg` of the source package:
- `pathspec >= 0.5.3` isn't used
- `setuptools` isn't an install requirements (it's a build requirement that should be defined in pyproject.toml. See https://setuptools.pypa.io/en/latest/userguide/quickstart.html)
- `pyeit` shouldn't be self referencing itself. You shouldn't import it in the `setup.py`.

I'd recommend you try to fix that in next release.
Again this isn't an issue with the conda recipe but with the source package.

Nothing blocking for this PR.

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/pyeit/meta.yaml (hunk 3 above)
Comment (liubenyuan) [reply recipes/pyeit/meta.yaml:27]: Thank you, These imports are removed from `setup.cfg` and I force updating the tag `v1.1.5` and the corresponding `sha256` in `meta.yaml`.

Comment (beenje) [reply recipes/pyeit/meta.yaml:27]: Overwriting a tag is really not recommended, especially as you pushed that version to pypi...
This package https://pypi.org/project/pyeit/1.1.5/ now doesn't match what you have in the git repo.

Always better to bump the version.

I'd suggest to put back the old tag and bump the version to 1.1.6.

Comment (liubenyuan) [reply recipes/pyeit/meta.yaml:27]: Ok, version 1.1.5 is reverted to the correct commit, and a new version 1.1.6 is bumped.

**Review by liubenyuan — COMMENTED** [review]
bump 1.1.6

**Pull request merged**